src/program.c

```diff
@@ -2451,6 +2454,15 @@ PlutoProg *scop_to_pluto_prog(osl_scop_p scop, PlutoOptions *options)
 
     PlutoProg *prog = pluto_prog_alloc();
 
+    /* Language */
+    if (!strcmp(scop->language, "f")) {
+        prog->language = CLOOG_LANGUAGE_FORTRAN;
+    } else if (!strcmp(scop->language, "p")) {
```

Comment:
No space before else. Similarly, at other places as well.

Reply:
Format `program.c` file with:
- 4 spaces indentation
- If a statement within a `for`, `while` or `if` is in the next line, surround it with braces.
- `if`, `while` and `for` of the form `INSTR (cond) {`
- `else` has no spaces between braces: `}else{`